org.freedesktop.Sdk.Extension.vala-stable.yml

```diff
@@ -0,0 +1,84 @@
+id: org.freedesktop.Sdk.Extension.vala-stable
```

Comment:
My only complaint is that it says `vala-stable`. I think just `vala` is enough, and then `vala-unstable` can be introduced if needed. It's considered stable by default unless stated otherwise.

Reply:
ok, I can do that.